packages/plugin-core/src/utils/ExtensionUtils.ts

```diff
@@ -313,7 +313,8 @@ export class ExtensionUtils {
       type: workspaceType,
       config: dendronConfig,
     } = workspace;
-    let numNotes = _.size(engine.notes);
+    const notes = await engine.findNotes({ excludeStub: false });
```

Comment:
How long does this take for org-workspace? We should change to `findNotesMeta` in as many places where possible.

Or in this case, just delete the telemetry for now until we have a better query model.

Reply:
Right now its still using `enginev2` so its reading from memory still. When we swap to v3, we should keep an eye on the time.